openupgrade_scripts/scripts/product/18.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,100 @@
+---Models in module 'product'---
+new model product.combo
+new model product.combo.item
+
+# NOTHING TO DO: moved in from pos
+
+new model update.product.attribute.value [transient]
+
+# NOTHING TO DO: new functionality
+
+---Fields in module 'product'---
+product      / product.attribute        / active (boolean)              : NEW hasdefault: default
+product      / product.attribute        / template_value_ids (one2many) : NEW relation: product.template.attribute.value
+product      / product.attribute.value  / active (boolean)              : NEW hasdefault: default
+product      / product.category         / message_follower_ids (one2many): NEW relation: mail.followers
+product      / product.category         / message_ids (one2many)        : NEW relation: mail.message
+product      / product.combo            / combo_item_ids (one2many)     : NEW relation: product.combo.item
+product      / product.combo            / company_id (many2one)         : NEW relation: res.company
+product      / product.combo            / name (char)                   : NEW required
+product      / product.combo            / sequence (integer)            : NEW hasdefault: default
+product      / product.combo.item       / combo_id (many2one)           : NEW relation: product.combo, required
+product      / product.combo.item       / company_id (many2one)         : NEW relation: res.company, isrelated: related, stored
+product      / product.combo.item       / extra_price (float)           : NEW hasdefault: default
+product      / product.combo.item       / product_id (many2one)         : NEW relation: product.product, required
+product      / product.document         / _order                        : _order is now 'sequence, name' ('name')
+
+# NOTHING TO DO
+
+product      / product.document         / sequence (integer)            : NEW hasdefault: default
+
+# DONE: create sequence matching name
+
+product      / product.pricelist        / _order                        : _order is now 'sequence, id, name' ('sequence asc, id asc')
+
+# NOTHING TO DO
+
+product      / product.pricelist        / discount_policy (selection)   : DEL required, selection_keys: ['with_discount', 'without_discount']
+product      / product.pricelist.item   / display_applied_on (selection): NEW required, selection_keys: ['1_product', '2_product_category'], hasdefault: default
```

Comment:
I have a BD with 342375 pricelist items. not sure if we should prefill this too.

Reply:
well measure it? In my experience, the impact is negligible for indexed columns on less than millions of rows tables